tools/prec_land.py

```diff
@@ -36,7 +36,6 @@
     )
     # print("sent", angle_x, angle_y, horizontal_fov, vertical_fov)
     time.sleep(1 / hertz)
-    print(time.time())
```

Comment:
keep this to know code is actually running, anyway, it doesn't run at `hertz`, `recv_match` is blocking so the `LANDING_TARGET` message is sent once `GLOBAL_POSITION_INT` is received (~1Hz)